x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/closure_options.test.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { ClosureOptions } from './closure_options';
+import { useMountAppended } from '../../../../utils/use_mount_appended';
+import { TestProviders } from '../../../../mock';
+import { ClosureOptionsRadio } from './closure_options_radio';
+
+describe('ClosureOptions', () => {
+  const mount = useMountAppended();
+
+  test('it renders', () => {
+    const wrapper = shallow(
+      <ClosureOptions
+        disabled={false}
+        closureTypeSelected="close-by-user"
+        onChangeClosureType={jest.fn()}
+      />
+    );
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  test('it shows the left side', () => {
```

Comment:
I would prefer the description to be a bit more straight forward, like its hows options from group